lib/solargraph/rbs_map/conversions.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'rbs'
```

Comment:
This was so I could get RBS class information into the Solargraph LSP as I made these changes - I can throw this in a `@!parse` block or something if leaving it out was intentional.